src/playback-watcher.js

```diff
@@ -51,7 +51,9 @@ export default class PlaybackWatcher {
     this.tech_.on('seekablechanged', fixesBadSeeksHandler);
     this.tech_.on('waiting', waitingHandler);
     this.tech_.on(timerCancelEvents, cancelTimerHandler);
-    this.monitorCurrentTime_();
+    if (this.tech_.hasStarted_ || this.tech_.isReady_) {
```

Comment:
If the tech is not ready by the time this constructor runs, this will continue to be broken. You should setup an event listener for the `ready` event instead of checking the private variables on the tech. The tech has a `this.tech_.ready()` function that will either setup a listener for when the `ready` event will be fired, or will run your callback immediately if the tech is already `ready`.